web/src/components/PoiListItem.tsx

```diff
@@ -74,16 +51,27 @@ const PoiListItem = ({ poi, distance, selectPoi }: PoiListItemProps): ReactEleme
 
   return (
     <>
-      <ListItemContainer id={slug}>
-        <LinkContainer onClick={selectPoi} tabIndex={0} label={title}>
-          <Thumbnail alt='' src={thumbnail || PoiThumbnailPlaceholder} />
-          <Description>
-            <Title>{title}</Title>
-            {distance !== null && <Distance>{t('distanceKilometre', { distance: distance.toFixed(1) })}</Distance>}
-            <Category>{category.name}</Category>
-          </Description>
-        </LinkContainer>
-      </ListItemContainer>
+      <ListItem disablePadding>
+        <StyledListItemButton onClick={selectPoi} id={slug} to={slug} component={Link} aria-label={title}>
+          <StyledListItemAvatar>
+            <Avatar src={thumbnail || PoiThumbnailPlaceholder} alt='' variant='square' />
+          </StyledListItemAvatar>
+          <StyledListItemText
+            disableTypography
+            primary={
+              <StyledTypography variant='title2' component='h2'>
+                {title}
+              </StyledTypography>
+            }
+            secondary={
+              <StyledTypography variant='body1' component='div'>
```

Comment:
What matches the design here is `body3` (fontSize: 12, fontWeight: 400)
You have to adjust the color to slightly gray like the design.
```suggestion
              <StyledTypography variant='body3' component='div'>
```

Reply:
hmm :thinking: should we stricktly follow the design? for me this font seems too small...

Reply:
this also looks good, so i would also go for it :+1: